### PR TITLE
Outscale SAS compute driver doc update and cleanup

### PR DESCRIPTION
**Pull request by jmunhoz**
First update and cleanup in the Outscale SAS compute driver documentation. The patches add the new Outscale logo and some modifications/improvements coming from the fellows in Outscale taking care of the final user documentation.

The patches also contribute some doc fixes in the parent and sibling classes (ec2.py) in order to generate coherent and robust documentation via herency.

Signed-off-by: Javier M. Mellid <jmunhoz@igalia.com>

### CODE REVIEW

File: docs/compute/drivers/outscale_sas.rst [hunk 1]
```diff
@@ -3,29 +3,30 @@ Outscale SAS Driver Documentation
 
 `Outscale SAS`_ provides an IaaS platform allowing
 developers to benefit from all the flexibility of the Cloud.
-This IaaS platform relies on TINA OS, its Cloud manager which purpose is to
+This IaaS platform relies on TINA OS, its Cloud manager whose purpose is to
 provide great performances on the Cloud.
-TINA OS is an own developed software with APIs compatible with AWS EC2 (TM).
+TINA OS is a software developed by Outscale with APIs compatible with AWS EC2 (TM).
```
Comment (tonybaloney) [inline docs/compute/drivers/outscale_sas.rst]: should be "is software developed by"

File: docs/compute/drivers/outscale_sas.rst [hunk 2]
```diff
@@ -3,29 +3,30 @@ Outscale SAS Driver Documentation
...
-* (Soon) Asia (Hong Kong) Region
+* eu-west-1, France (Paris)
+* eu-west-2, France (Paris)
 
-Outscale SAS is a European company: prices are in €.
+For other Regions, see the `Outscale Inc.`_ documentation.
 
-API Docs
---------
+Outscale SAS is European company and uses prices in Euros.
```
Comment (tonybaloney) [inline docs/compute/drivers/outscale_sas.rst]: "and is priced in Euros"

**Review by tonybaloney — COMMENTED** [review]
couple of small changes, thanks for the contribution

File: docs/compute/drivers/outscale_sas.rst (hunk 1 above)
Comment (jmunhoz) [reply docs/compute/drivers/outscale_sas.rst]: yep, thanks!

File: docs/compute/drivers/outscale_sas.rst (hunk 2 above)
Comment (jmunhoz) [reply docs/compute/drivers/outscale_sas.rst]: good catch, thanks!

**Comment by jmunhoz** [comment]
@tonybaloney I am pushing some new doc patches here but I am waiting for some possible new patches and the final confirmation coming from the translation team in Outscale, so feel free to put on hold this PR review till this confirmation is available. Thanks!

**Comment by tonybaloney** [comment]
@jmunhoz did that happen?

**Comment by jmunhoz** [comment]
@tonybaloney let me confirm with the Outscale fellows. I hope to have more information this week. Thanks!

**Comment by jmunhoz** [comment]
@tonybaloney yes, it happened. feel free to merge. thanks!

**Comment by tonybaloney** [comment]
:+1:

**Pull request closed without merging**